module/modmanager/manager.go

```diff
@@ -1067,7 +1068,7 @@ func (m *module) dial() error {
 	var err error
 	addrToDial := m.addr
 	if !rutils.TCPRegex.MatchString(addrToDial) {
-		addrToDial = "unix://" + m.addr
+		addrToDial = "unix://" + cleanWindowsSocketPath(addrToDial)
```

Comment:
I see that the `m.addr` variable is "already cleaned" during [this assignment](https://github.com/viamrobotics/rdk/pull/4878/files#diff-a4308b77f384da81096eda21bf023471e80818dda86e74f9335f99a6a57d07c9R1202).

Did we observe more cleaning was needed here? Or is this just defensive?

Reply:
wasn't thinking about the full history of that member

good catch, removed